src/js/components/FormField.js

```diff
@@ -54,55 +55,46 @@ export default class FormField extends Component {
   }
 
   render () {
-    let classes = [CLASS_ROOT];
-    if (this.state.focus) {
-      classes.push(CLASS_ROOT + "--focus");
-    }
-    if (this.props.hidden) {
-      classes.push(CLASS_ROOT + "--hidden");
-    }
-    if (this.props.htmlFor) {
-      classes.push(CLASS_ROOT + "--text");
-    }
-    if (this.props.size) {
-      classes.push(CLASS_ROOT + "--size-" + this.props.size);
-    }
-    if (this.props.strong) {
-      classes.push(`${CLASS_ROOT}--strong`);
-    }
-    if (this.props.className) {
-      classes.push(this.props.className);
-    }
+    const { 
+      children, className, help, hidden, htmlFor, label, size, strong, error, 
+      ...props
+    } = this.props;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      {
+        [`${CLASS_ROOT}--focus`]: this.state.focus,
+        [`${CLASS_ROOT}--hidden`]: hidden,
+        [`${CLASS_ROOT}--text`]: htmlFor,
+        [`${CLASS_ROOT}--size-${size}`]: size,
+        [`${CLASS_ROOT}--strong`]: strong,
+        [`${CLASS_ROOT}--error`]: error
+      },
+      className
+    );
 
-    let error;
-    if (this.props.error) {
-      classes.push(CLASS_ROOT + "--error");
-      error = (
-        <span className={CLASS_ROOT + "__error"}>{this.props.error}</span>
-      );
-    }
-    let help;
-    if (this.props.help !== null && this.props.help !== undefined) {
-      help = <span className={CLASS_ROOT + "__help"}>{this.props.help}</span>;
-    }
+    const fieldError = (error)
+      ? <span className={CLASS_ROOT + "__error"}>{error}</span>
+      : null;
```

Comment:
A small point, our convention for these situations is to use `undefined` instead of `null`.